golang/lib/services/service.go

```diff
@@ -21,3 +21,24 @@ package services
 The identifier used for services with the network.
 */
 type ServiceID string
+
+
+type SharedDirectory struct {
+	//The service directory absolute path in the testsuite container
```

Comment:
I'd say "in the container where this code is running", since it could actually be running in a Lambda (not a testsuite)